src/main/java/uk/gov/companieshouse/model/dto/email/ApplicationAcceptedEmailData.java

```diff
@@ -8,8 +8,6 @@
 
     private String companyName;
 
-    private String paymentReference;
```

Comment:
Why are we removing the paymentReference from the email? We just need to stop sending the email to finance. The payment reference needs to stay.